app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user, class_name: 'User', foreign_key: 'user_id'
+  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'

```

Comment:
[OPTIONAL] Rails adds foreign keys as default. So no need for adding foreign keys for other associations except the associations to the author_id column. So kindly consider removing the foreign_keys here.